doc/OnlineDocs/source/concepts/ruc_sced_cycle.rst

```diff
@@ -1,42 +1,69 @@
 The Prescient Simulation Cycle
-=============================================
+==============================
 
-.. note::
+.. image:: ../_static/image/RucScedCycle.png
+	:class: align-right
 
-   This was taken from a previous write-up and needs to be revisited.
+Prescient simulates the operation of a power generation network throughout a 
+study horizon, finding the set of operational choices that satisfy demand at
+the lowest possible cost.
 
-Prescient simulates the operation of the network throughout a study horizon, finding
-the set of operational choices that satisfy demand at the lowest possible cost.
+A Prescient simulation consists of two repeating cycles, one nested in the
+other. The outer cycle is the Reliability Unit Commitment (RUC) planning
+cycle, which schedules changes in dispatchable generators' online status during
+the cycle's period. The inner, more frequent cycle is the Security Constrained
+Economic Dispatch (SCED) cycle, which determines dispatch levels for
+dispatchable generators.
 
-Prescient loops through two repeating phases, the reliability unit commitment (RUC) phase and 
-the security constrained economic dispatch (SCED) phase.  The RUC phase determines which
-dispatchable generators will be active in upcoming operational time periods. For each operational
-period within a RUC cycle, the SCED phase selects the dispatch level of each committed thermal 
-generator.
+The RUC Cycle
+-------------
 
-The RUC phase occurs one or more times per day.  Each time the RUC phase occurs, Prescient generates
-a unit commitment schedule that indicates which generators will be brought online or taken offline
-within the RUC's time horizon. The SCED phase occurs one or more times per hour.  Each SCED selects
-a thermal dispatch level for each committed generator.
+The RUC cycle periodically generates a RUC plan. A RUC plan consists of two types of
+data: a unit commitment schedule and a pricing schedule. The unit commitment schedule

```

Comment:
In order to compute day-ahead pricing data, the option `compute_market_settlements` must be set to `True` (the default is `False`).